internal/registry/buildpack.go

```diff
@@ -2,18 +2,21 @@ package registry
 
 import (
 	"fmt"
+	"strings"
 
 	ggcrname "github.com/google/go-containerregistry/pkg/name"
 	"github.com/pkg/errors"
+
+	"github.com/buildpacks/pack/internal/style"
 )
 
 // Buildpack contains information about a buildpack stored in a Registry
 type Buildpack struct {
 	Namespace string `json:"ns"`
 	Name      string `json:"name"`
 	Version   string `json:"version"`
-	Yanked    bool   `json:"yanked"`
-	Address   string `json:"addr"`
+	Yank      bool   `json:"yanked"`
```

Comment:
Not a super strong opinion but I think this was better off as the "state" of what was desired instead of attempting to describe a desired operation.

```suggestion
	Yanked      bool   `json:"yanked"`
```

Reply:
@jromero I have been torn on this myself - good to have your input.  I'll makes some tweaks.